static/src/javascripts/projects/common/modules/experiments/tests/participation-low-fric-music-v1.js

```diff
@@ -0,0 +1,129 @@
+define([
+    'common/utils/config',
+    'common/utils/mediator',
+    'Promise',
+    'common/utils/detect',
+    'common/modules/experiments/low-friction-participation'
+], function (
+    config,
+    mediator,
+    Promise,
+    detect,
+    lowFrictionParticipation
+) {
+
+    var omniture,
+        isAGigReview,
+        isAnAlbumReview;
+
+    /**
+     * The omniture module depends on common/modules/experiments/ab, so trying to
+     * require omniture directly inside an AB test gives you a circular dependency.
+     *
+     * This is a workaround to load omniture without making it a dependency of
+     * this module, which is required by an AB test.
+     */
+    function getOmniture() {
+        return new Promise(function (resolve) {
+            if (omniture) {
+                return resolve(omniture);
+            }
+
+            require('common/modules/analytics/omniture', function (omnitureM) {
```

Comment:
Will this actually work outside of dev? I didn't think this would be exposed as its own bundle in prod.